internify/src/components/molecules/RadioButtonsGroup.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import RadioTemp from '../atoms/RadioTemp';
+import FormControl from '@material-ui/core/FormControl';
+
+function RadioButtonsGroup() {
```

Comment:
If we're making grouped components for a particular page, we can give it a more specific naming scheme and group them in a sub-folder inside of molecules. I'm thinking we can call this component `WorkingExperienceRadio` inside of a folder called `createJobHeader` or something

Reply:
so you want all components related to a page under the folder createJobHeader?

Reply:
Yeah basically! `CreateJobHeader` can be an organism since it is a large component that we will use in the create job header page